### PR TITLE
Coverage test: coins tests 

### PR DESCRIPTION
**Pull request by hieuvubk**
This Pr is to add the test case in issue https://github.com/Pylons-tech/pylons/issues/497.
Now coins tests available, I'll write more test for another tasks

### CODE REVIEW

**Review by faddat — APPROVED** [review]

File: x/pylons/keeper/coins_test.go [hunk 1]
```diff
@@ -47,3 +47,126 @@ func (suite *IntegrationTestSuite) TestMintCoinsToAddr() {
 	balance := bk.SpendableCoins(ctx, addr)
 	require.True(balance.IsEqual(mintAmt))
 }
+
+func (suite *IntegrationTestSuite) TestBurnCoins() {
+	// k, ctx := setupKeeper(t)
```
Comment (mustafapylons) [inline x/pylons/keeper/coins_test.go:52]: Can we remove dead code?

File: x/pylons/keeper/coins_test.go [hunk 2]
```diff
@@ -47,3 +47,126 @@ func (suite *IntegrationTestSuite) TestMintCoinsToAddr() {
...
+	err = k.BurnCoins(ctx, types.PaymentsProcessorName, mintAmt)
+	require.NoError(err)
+
+	addr := k.CoinsIssuerAddress()
+	balance := bk.SpendableCoins(ctx, addr)
+	require.True(balance.IsEqual(sdk.Coins{}))
+}
+
+func (suite *IntegrationTestSuite) TestBurnCoinsFromAddress() {
+	// k, ctx := setupKeeper(t)
```
Comment (mustafapylons) [inline x/pylons/keeper/coins_test.go:76]: This dead code as well

File: x/pylons/keeper/coins_test.go [hunk 3]
```diff
@@ -47,3 +47,126 @@ func (suite *IntegrationTestSuite) TestMintCoinsToAddr() {
...
+	require.NoError(err)
+
+	err = k.BurnCreditFromAddr(ctx, addr, mintAmt)
+
+	balance := bk.SpendableCoins(ctx, addr)
+	require.True(balance.IsEqual(sdk.Coins{}))
+}
+
+func (suite *IntegrationTestSuite) TestMintCreditToAddr() {
+	// k, ctx := setupKeeper(t)
```
Comment (mustafapylons) [inline x/pylons/keeper/coins_test.go:99]: This dead code can be removed as well

File: x/pylons/keeper/coins_test.go [hunk 4]
```diff
@@ -47,3 +47,126 @@ func (suite *IntegrationTestSuite) TestMintCoinsToAddr() {
...
+	feeCollectorBalances := bk.SpendableCoins(ctx, feeCollectorAddr)
+	userBalance := bk.SpendableCoins(ctx, addr)
+
+	require.True(userBalance.IsEqual(mintCoins.Sub(burnCoins).Sub(feesCoins)))
+	require.True(processorBalances.IsEqual(sdk.Coins{}))
+	require.True(feeCollectorBalances.IsEqual(feesCoins))
+}
+
+func (suite *IntegrationTestSuite) TestSendRewardsFromFeeCollector() {
+	// k, ctx := setupKeeper(t)
```
Comment (mustafapylons) [inline x/pylons/keeper/coins_test.go:136]: This dead code can be removed as well.

**Review by mustafapylons — COMMENTED** [review]
Few comments, otherwise code is good.

**Review by faisalnaveedRNS — APPROVED** [review]

**Review by mustafapylons — COMMENTED** [review]
One more typo, otherwise it's good.

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Pull request merged**